+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ grep expect
+ rpm -qa
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n '' ']'
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Fri Aug 18 09:50:43 2023 - INFO  - Start testing...
+ lsblk
++ sed -n '2, 1p' lsblk.txt
++ awk '{print $1}'
+ dev=sr0
+ hdparm -h
oe_test_hdparm.sh: line 27: hdparm: command not found
+ grep -i Usage
+ CHECK_RESULT 1 0 0 'hdparm -h fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='hdparm -h fail'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'hdparm -h fail'
+ LOG_ERROR 'hdparm -h fail'
+ message='hdparm -h fail'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'hdparm -h fail'
Fri Aug 18 09:50:43 2023 - ERROR - hdparm -h fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_hdparm.sh line 28'
+ message='oe_test_hdparm.sh line 28'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_hdparm.sh line 28'
Fri Aug 18 09:50:43 2023 - ERROR - oe_test_hdparm.sh line 28
+ return 0
+ hdparm -a /dev/sr0
oe_test_hdparm.sh: line 29: hdparm: command not found
+ grep -i /dev/sr0
+ CHECK_RESULT 1 0 0 'hdparm -a system fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='hdparm -a system fail'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'hdparm -a system fail'
+ LOG_ERROR 'hdparm -a system fail'
+ message='hdparm -a system fail'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'hdparm -a system fail'
Fri Aug 18 09:50:43 2023 - ERROR - hdparm -a system fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_hdparm.sh line 30'
+ message='oe_test_hdparm.sh line 30'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_hdparm.sh line 30'
Fri Aug 18 09:50:43 2023 - ERROR - oe_test_hdparm.sh line 30
+ return 0
+ hdparm -b /dev/sr0
oe_test_hdparm.sh: line 31: hdparm: command not found
+ grep -i /dev/sr0
+ CHECK_RESULT 1 0 0 'hdparm -b system fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='hdparm -b system fail'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'hdparm -b system fail'
+ LOG_ERROR 'hdparm -b system fail'
+ message='hdparm -b system fail'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'hdparm -b system fail'
Fri Aug 18 09:50:44 2023 - ERROR - hdparm -b system fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_hdparm.sh line 32'
+ message='oe_test_hdparm.sh line 32'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_hdparm.sh line 32'
Fri Aug 18 09:50:44 2023 - ERROR - oe_test_hdparm.sh line 32
+ return 0
+ hdparm -c /dev/sr0
oe_test_hdparm.sh: line 33: hdparm: command not found
+ grep -i IO_support
+ CHECK_RESULT 1 0 0 'hdparm -c system fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='hdparm -c system fail'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'hdparm -c system fail'
+ LOG_ERROR 'hdparm -c system fail'
+ message='hdparm -c system fail'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'hdparm -c system fail'
Fri Aug 18 09:50:44 2023 - ERROR - hdparm -c system fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_hdparm.sh line 34'
+ message='oe_test_hdparm.sh line 34'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_hdparm.sh line 34'
Fri Aug 18 09:50:44 2023 - ERROR - oe_test_hdparm.sh line 34
+ return 0
+ hdparm -m /dev/sr0
oe_test_hdparm.sh: line 35: hdparm: command not found
+ grep -i multcount
+ CHECK_RESULT 1 0 0 'hdparm -m system fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='hdparm -m system fail'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'hdparm -m system fail'
+ LOG_ERROR 'hdparm -m system fail'
+ message='hdparm -m system fail'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'hdparm -m system fail'
Fri Aug 18 09:50:44 2023 - ERROR - hdparm -m system fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_hdparm.sh line 36'
+ message='oe_test_hdparm.sh line 36'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_hdparm.sh line 36'
Fri Aug 18 09:50:44 2023 - ERROR - oe_test_hdparm.sh line 36
+ return 0
+ hdparm -V /dev/sr0
oe_test_hdparm.sh: line 37: hdparm: command not found
+ grep -i hdparm
+ CHECK_RESULT 1 0 0 'hdparm -V system fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='hdparm -V system fail'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'hdparm -V system fail'
+ LOG_ERROR 'hdparm -V system fail'
+ message='hdparm -V system fail'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'hdparm -V system fail'
Fri Aug 18 09:50:44 2023 - ERROR - hdparm -V system fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_hdparm.sh line 38'
+ message='oe_test_hdparm.sh line 38'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_hdparm.sh line 38'
Fri Aug 18 09:50:44 2023 - ERROR - oe_test_hdparm.sh line 38
+ return 0
+ hdparm -r /dev/sr0
oe_test_hdparm.sh: line 39: hdparm: command not found
+ grep -i readonly
+ CHECK_RESULT 1 0 0 'hdparm -r system fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='hdparm -r system fail'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'hdparm -r system fail'
+ LOG_ERROR 'hdparm -r system fail'
+ message='hdparm -r system fail'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'hdparm -r system fail'
Fri Aug 18 09:50:44 2023 - ERROR - hdparm -r system fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_hdparm.sh line 40'
+ message='oe_test_hdparm.sh line 40'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_hdparm.sh line 40'
Fri Aug 18 09:50:44 2023 - ERROR - oe_test_hdparm.sh line 40
+ return 0
+ hdparm -I /dev/sr0
oe_test_hdparm.sh: line 41: hdparm: command not found
+ grep -i Capabilities
+ CHECK_RESULT 1 0 0 'hdparm -I system fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='hdparm -I system fail'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'hdparm -I system fail'
+ LOG_ERROR 'hdparm -I system fail'
+ message='hdparm -I system fail'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'hdparm -I system fail'
Fri Aug 18 09:50:44 2023 - ERROR - hdparm -I system fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_hdparm.sh line 42'
+ message='oe_test_hdparm.sh line 42'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_hdparm.sh line 42'
Fri Aug 18 09:50:44 2023 - ERROR - oe_test_hdparm.sh line 42
+ return 0
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Fri Aug 18 09:50:44 2023 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 8
+ test 8 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri Aug 18 09:50:44 2023 - ERROR - failed to execute the case.
+ exit 8
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Fri Aug 18 09:50:44 2023 - INFO  - start environment cleanup.
+ rm -rf lsblk.txt
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Fri Aug 18 09:50:44 2023 - INFO  - Finish environment cleanup!
